challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  return input.map do |n| n.capitalize end
```

Comment:
Rubyists tend to prefer curly braces (`{}`) for single line blocks.

https://github.com/bbatsov/ruby-style-guide#single-line-blocks